README.md

```diff
@@ -124,4 +126,12 @@ To start the registry inside the virtualenv that `pipenv` creates:
 FLASK_APP=app.py pipenv run flask run
 ```
 
-Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
\ No newline at end of file
+Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
+
+You can then navigate to http://localhost:5000/admin in the browser.
+
+### Debugging
+
+If you are served an error message on the admin home screen, you may need to run `npm install` in the root directory of the library_registry repo. If you're running the application locally, you can also try running the same command in the root directory of the registry_admin repo.
```

Comment:
You should add that these steps are for when you are running the app locally without Docker. For the first sentence, for production instances it's most likely the case that the app is running with Docker, so maybe you can add something that says to also look at the Docker configuration if that's what internally runs `npm install`.

Reply:
Fixed!